docs/pages/client_usage.mdx

```diff
@@ -0,0 +1,237 @@
+
+
+## Additional Flags
+
+The Pelican client support a variety of command line flags that modify the client's behavior:
+
+### Global Flags:
+
+- **-h or --help:** Takes no argument and can be used with any Pelican sub command for more information about the sub command and additional supported flags.
+- **-f or --federation:** Takes a URL that indicates to Pelican which federation the request should be made to.
+- **-d or --debug:** Takes no argument, but runs Pelican in debug mode, which provides verbose output for debugging purposes.
+- **--config:** Takes a filepath and indicates to Pelican the location of the Pelican configuration file.
+- **--json:** Takes no argument and outputs results in JSON format.
+- **-l or --log:** Takes a string that specifies a file location to output the pelican logs to rather than the stderr.
+- **--version:** Prints the version of Pelican and exits.
+
+### Flags For `object get/put/copy`:
+
+- **-c or --cache:** Takes a cache URL and indicates to Pelican that only the specified cache should be used. When used, Pelican will not attempt to use other caches if the provided cache cannot provide the file.
+- **--caches:** Takes the path to a JSON file containing a list of caches. Similar to the `-c` flag, Pelican will attempt to use only these caches in the order they are listed.
+- **-h or --help:** Gives additional information on how to use the command as well as lists these flags with short descriptions for the `object copy` command.
+- **--methods:** Takes a comma seperated list of methods to try for downloads/uploads, the default is just http.
+- **-r or --recursive:** Takes no argument and indicates to Pelican that all sub paths at the level of the provided namespace should be copied recursively. This option is only supported if the origin supports the WebDav protocol.
+- **-t or --token:** Takes a path to a file containing a signed JWT, and is used to download protected objects.
+
+## Aliases of The Pelican Binary
+
+The Pelican binary can change its behavior depending on what it is named. This feature serves two purposes; it allows Pelican to use a few convenient default settings in the case that the federation being interacted with is the OSDF, and it allows Pelican to run in legacy `stashcp` and `stash_plugin` modes.
+
+### Prefixing The Binary With OSDF
+
+When the name of the Pelican binary begins with `osdf`, Pelican will assume that all objects are coming from the OSDF which allows it to make several assumptions. The most immediate effect for users is that you no longer need to specify a federation URL, ONLY with no URL scheme, `osdf:///`, or `stash:///` URL schemes. The command to download a public file from above can then be simplified to:
+
+```bash
+osdf object copy osdf:///ospool/uc-shared/public/OSG-Staff/validation/test.txt downloaded-testfile.txt
```

Comment:
This can be simplified further to `osdf object copy /ospool/uc-shared/public/OSG-Staff/validation/test.txt downloaded-testfile.txt`, can't it? The `osdf://` scheme does the same thing when using the regular pelican client, so there's not a whole lot of benefit to naming the binary `osdf` unless we tell the user they can stop using ANY url-scheme stuff.

Reply:
You are correct, having no scheme will do the same as having an `osdf://` scheme. I'll leave the example scheme blank to show users it can be simplified even further.